src/app/components/header/header.component.html

```diff
@@ -0,0 +1,38 @@
+<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
+  <a class="navbar-brand" href="#">Bestapp</a>
+  <button class="navbar-toggler"
+          type="button"
+          (click)="toggleNavbar()">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+
+  <div class="collapse navbar-collapse"
+       [ngClass]="{ 'show': show }">
+    <ul class="navbar-nav mr-auto">
+      <li class="nav-item">
+        <a routerLink="home" routerLinkActive="active" class="nav-link" (click)="closeOnTransit()">Home</a>
+      </li>
+      <li class="nav-item">
+        <a routerLink="two-way-binding" routerLinkActive="active" class="nav-link" (click)="closeOnTransit()">lr1</a>
+      </li>
+      <li class="nav-item">
+        <a routerLink="game-control" routerLinkActive="active" class="nav-link" (click)="closeOnTransit()">lr2</a>
+      </li>
+<!--      <li class="nav-item dropdown">-->
+<!--        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">-->
+<!--          Dropdown-->
```

Comment:
Commented code don't need to be in PR

Reply:
I'll fix it in further task branch(laboratory 3)